src/Providers/UrbanAirshipV3.php

```diff
@@ -237,6 +238,14 @@ public function validateBeforePush()
             throw new MissingArgumentException('You have to setMessage() before sending push');
         }
 
+        if (!empty($this->getNamedUsers()) AND !empty($this->getAliases())) {
+            throw new InvalidAudienceException('Send push to either named users or aliases - not both.');
+        }
+
+        if (in_array('wns', $this->getPlatforms()) AND !empty($this->getNamedUsers())) {
```

Comment:
can we change AND to || to keep the code style of rest of the plugin

Reply:
Sure. Bad old habits on my part. `&&` and `||` are safer in all cases than `AND` and `OR`